src/Fixer/LanguageConstruct/ClassKeywordRemoveFixer.php

```diff
@@ -159,7 +159,7 @@ private function replaceClassKeyword(Tokens $tokens, $classIndex)
         $classBeginIndex = $classEndIndex;
         while ($tokens[--$classBeginIndex]->isGivenKind(array(T_NS_SEPARATOR, T_STRING)));
         ++$classBeginIndex;
-        $classString = $tokens->generatePartialCode($classBeginIndex, $classEndIndex);
+        $classString = ltrim($tokens->generatePartialCode($classBeginIndex, $classEndIndex), '\\');
```

Comment:
I would rather move the `$classBeginIndex` if its `T_NS_SEPARATOR` than creating bigger partial code and then shrink it back.